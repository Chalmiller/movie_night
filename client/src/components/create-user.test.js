import React from 'react';
import { shallow } from 'enzyme';
import CreateUser from './create-user.component';

it('renders create new user without crashing', () => {
  expect(shallow(<CreateUser/>)).toMatchSnapshot();
});