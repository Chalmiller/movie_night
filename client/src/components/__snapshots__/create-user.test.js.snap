// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders create new user without crashing 1`] = `ShallowWrapper {}`;
